### ISSUE TITLE
Problems encountered in calculating CSG

### ISSUE THREAD

**Issue by wangsuny** [Dref360/spectral_metric#24]
Hello,I am sorry to disturb you. 
I am a student majoring in chemical engineering. I found your research very helpful to me. Thanks a lot. But when I try to figure out the CSG of CIFAR-10 loaded from keras. I cannot obtain the result mentioned in your article(1.0), My result is 3.53361029. I am looking forward to receiving your help very much, thanks a lot again. This is my code:
from spectral_metric.estimator import CumulativeGradientEstimator
from keras.datasets import cifar10
(train_images,train_labels),(test_images,test_labels) = cifar10.load_data()
train_images =train_images/255
train_images = train_images.reshape((train_images.shape[0],-1))
train_labels = train_labels.reshape([-1])
estimator = CumulativeGradientEstimator()
estimator.fit(train_images,train_labels)
csg = estimator.csg
print(csg)

**Comment by Dref360** [comment]
Hello!

Thank you for submitting your issue, I'm happy to see that this is useful.

The CSG of 1.0 for Cifar10 is when using CNN autoencoder features and t-sne. 

Table 4 in Annex has all the values for all the configuration.

To get the autoencoder features [Link](https://github.com/Dref360/spectral-metric/blob/master/experiments/embedding/cnn_autoencoder.py)
To get the T-SNE features [Link](https://github.com/Dref360/spectral-metric/blob/master/experiments/embedding/tsne.py)


I hope this helps and I'm happy to hop on a call if it's easier.

**Comment by wangsuny** [comment]
Thank you very much for your help. I am working hard to achieve it.
